### PR TITLE
l10n: Add word "Name"

### PR DESCRIPTION
**Pull request by Valdnet**
Fix issue #2430.

Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

File: src/js/components/Poll/PublicRegisterModal.vue [hunk 1]
```diff
@@ -190,13 +190,13 @@ export default {
 
 			if (!this.isValidName) {
 				return {
-					result: t('polls', '{username} is invalid or reserved.', { username: this.userName }),
+					result: t('polls', 'Name {username} is invalid or reserved.', { username: this.userName }),
```
Comment (dartcafe) [inline src/js/components/Poll/PublicRegisterModal.vue:193]: "Name Peter is invalid" sounds strange to me. Either it needs an article ("The name Peter is invalid") or the term stays as before ("Peter is invalid").

File: src/js/components/Poll/PublicRegisterModal.vue [hunk 2]
```diff
@@ -190,13 +190,13 @@ export default {
...
 					status: 'error',
 				}
 			}
 
 			return {
-				result: t('polls', '{username} is valid.', { username: this.userName }),
+				result: t('polls', 'Name {username} is valid.', { username: this.userName }),
```
Comment (dartcafe) [inline src/js/components/Poll/PublicRegisterModal.vue:199]: Dito. In doubt we could remove this and the 'Checking name ...' as well, return an empty string as default and skip the information. This reduces the noise.

**Review by dartcafe — CHANGES_REQUESTED** [review]

File: src/js/components/Poll/PublicRegisterModal.vue (hunk 1 above)
Comment (rakekniven) [reply src/js/components/Poll/PublicRegisterModal.vue:193]: "User name **Peter** is invalid" would be my suggestion.

File: src/js/components/Poll/PublicRegisterModal.vue (hunk 2 above)
Comment (rakekniven) [reply src/js/components/Poll/PublicRegisterModal.vue:199]: Where does this message appears? As a Pop up? Or below a form item?

Comment (Valdnet) [reply src/js/components/Poll/PublicRegisterModal.vue:199]: > In doubt we could remove this and the 'Checking name ...' as well, return an empty string as default and skip the information. This reduces the noise.

@dartcafe What about a wrong username?

> Where does this message appears?

@rakekniven See issue # 2430.

**Review by rakekniven — APPROVED** [review]

File: src/js/components/Poll/PublicRegisterModal.vue (hunk 1 above)
Comment (dartcafe) [reply src/js/components/Poll/PublicRegisterModal.vue]: @rakekniven We had feedback, that the term `username` could be misleading for users who visit a public poll and are also a registered user of this site. This was the initial reason, why the term `username` was removed.

File: src/js/components/Poll/PublicRegisterModal.vue [hunk 3]
```diff
@@ -190,13 +190,13 @@ export default {
 
 			if (!this.isValidName) {
 				return {
-					result: t('polls', 'Name {username} is invalid or reserved.', { username: this.userName }),
+					result: t('polls', 'User name {username} is invalid or reserved.', { username: this.userName }),
```
Comment (dartcafe) [inline src/js/components/Poll/PublicRegisterModal.vue:193]: Please see my last comment. User name or username should be avoided in this combination.

**Review by dartcafe — CHANGES_REQUESTED** [review]

File: src/js/components/Poll/PublicRegisterModal.vue (hunk 2 above)
Comment (dartcafe) [reply src/js/components/Poll/PublicRegisterModal.vue]: > What about a wrong username? 

@Valdnet See also my last [comment](https://github.com/nextcloud/polls/pull/2443#discussion_r884582223).

File: src/js/components/Poll/PublicRegisterModal.vue (hunk 1 above)
Comment (rakekniven) [reply src/js/components/Poll/PublicRegisterModal.vue]: Ok, good catch.

So what about "Participant name **Peter** is invalid or reserved"?

File: src/js/components/Poll/PublicRegisterModal.vue (hunk 2 above)
Comment (rakekniven) [reply src/js/components/Poll/PublicRegisterModal.vue]: See my comment above.

File: src/js/components/Poll/PublicRegisterModal.vue (hunk 3 above)
Comment (Valdnet) [reply src/js/components/Poll/PublicRegisterModal.vue:193]: @dartcafe So it should be: `User name is invalid or reserved.`?

Comment (dartcafe) [reply src/js/components/Poll/PublicRegisterModal.vue:193]: @Valdnet 

Username can be interpreted as a user id, which is the login name of a user. In this case we just ask for a random name, therefore is should only be called `name`, IMHO.

It is just "Name Peter is invalid or reserved." still sounds strange to me, why I prefer to say "The name Peter is invalid or reserved."

Comment (dartcafe) [reply src/js/components/Poll/PublicRegisterModal.vue:193]: That's what I think fits best, if the old variant isn't clear enough fot the translations.

**Review by rakekniven — APPROVED** [review]

**Pull request merged**